### ISSUE TITLE
Uncaught module code-push./utils/code-push-error not found

### ISSUE THREAD

**Issue by sandrolombardo** [microsoft/cordova-plugin-code-push#610]
### Description

Since yesterday my cordova app didn't start anymore. App crashes caused by a missing module imported by code-push plugin. Console log prints out following stacktrace:

```
cordova.js:59 Uncaught module code-push./utils/code-push-error not found
require @ cordova.js:59
localRequire @ cordova.js:49
(anonymous) @ acquisition-sdk.js:3
build @ cordova.js:53
require @ cordova.js:68
localRequire @ cordova.js:49
exports.mapModules @ cordova.js:1340
(anonymous) @ cordova.js:1138
f @ cordova.js:649
Channel.fire @ cordova.js:777
(anonymous) @ cordova.js:1130
onScriptLoadingComplete @ cordova.js:1567
scriptLoadedCallback @ cordova.js:1584
(anonymous) @ cordova.js:1538
load (async)
exports.injectScript @ cordova.js:1524
injectIfNecessary @ cordova.js:1536
handlePluginsObject @ cordova.js:1589
(anonymous) @ cordova.js:1618
(anonymous) @ cordova.js:1538
load (async)
exports.injectScript @ cordova.js:1524
injectIfNecessary @ cordova.js:1536
exports.load @ cordova.js:1616
(anonymous) @ cordova.js:1129
setTimeout (async)
(anonymous) @ cordova.js:1128
build @ cordova.js:53
require @ cordova.js:68
(anonymous) @ cordova.js:1861
(anonymous) @ cordova.js:1863
```

I recognized that code-push released a new version 3.1.3 yesterday (20th of april 2020). In this new plugin version has been added following new import line in file "acquisition-sdk.ts": 

`import { CodePushHttpError, CodePushDeployStatusError, CodePushPackageError } from "../utils/code-push-error"`

On the code-push plugin side, the implementation seems to be correct. When building the platforms by Cordova, the subfolder ".../plugins/code-push/script/" is generated, but not the subfolder ".../plugins/code-push/utils/", which contains the file which is newly imported by "acquisition-sdk.ts".

This probably leads to the problem mentioned above.

### Reproduction

1. create new Cordova project using: cordova create hello com.example.hello HelloWorld
2. add ios platform (from project root of course): cordova platform add browser (or ios)
3. install code-push: cordova plugin add cordova-plugin-code-push
4. run project on emulator: cordova run browser (or ios)
5. Cordova should display green badge that "Device is ready"

Expected result:
Cordova should show "Device is ready" as before

Actual result:
Cordova is displaying error in developer console and "Connecting to device" message

### Additional Information

* cordova-plugin-code-push version: 
1.12.2

* List of installed plugins: 
code-push 3.1.3 "CodePushAcquisition"
cordova-plugin-code-push 1.12.2 "CodePush"
cordova-plugin-compat 1.2.0 "Compat"
cordova-plugin-device 2.0.3 "Device"
cordova-plugin-dialogs 2.0.2 "Notification"
cordova-plugin-file 4.3.3 "File"
cordova-plugin-file-transfer 1.6.3 "File Transfer"
cordova-plugin-whitelist 1.3.4 "Whitelist"
cordova-plugin-zip 3.1.0 "cordova-plugin-zip"

* Cordova version: 
6.5.0

* iOS/Android/Windows version: 
iOS 13.3.1, but it doesn't work also other systems (emulating in browser for example on Windows 10)

* Does this reproduce on a debug build or release build?
Didn't check

* Does this reproduce on a simulator, or only on a physical device?
On a simulator and also on a physical device

**Comment by mak4naki** [comment]
Hi, same problem fo me.
I think the problem is that utils/code-push-error.js is not copied inside cordova build folder

i solved using previous version of the plugin code-push
code-push 3.1.2 "CodePushAcquisition"

**Comment by ermik** [comment]
This is a `code-push` module dependency

@alexandergoncharov please close this in favor of microsoft/code-push#729

**Comment by andreidubov** [comment]
Hi @sandrolombardo 
Thanks for your find.
We fixed this issue in code-push v4.0.1

Please let me know if you have any questions.
